keyboards/zinc/keymaps/via/keymap.c

```diff
@@ -0,0 +1,195 @@
+#include QMK_KEYBOARD_H
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+rgblight_config_t RGB_current_config;
```

Comment:
These don't actually seem to be used in any real way. 

```suggestion
```

Reply:
Yes, these are old codes.
Thanks!